cypress/integration/e2e/homepage_spec.js

```diff
@@ -0,0 +1,94 @@
+describe('Home page tests', function()
```

Comment:
I do believe we have babel in this app which means we can use arrow functions, which is the standard we use in MAST. I also think it looks much neater. Example `describe('Home page tests', () => {...})`, this can be done with all the `function()` calls on this page.

Reply:
I completely forgot about this.
The only reason I wrote it like this was that I was following the Cypress tutorials and this is how they write it, but if the best practice is to write it like that then I will go through them all and change it to all be aligned 👍